overlord/snapstate/snapstate_test.go

```diff
@@ -7273,8 +7279,10 @@ func (s *snapmgrTestSuite) TestUpdateOneAutoAliasesScenarios(c *C) {
 
 		tasks := ts.Tasks()
 		// make sure the last task from Update is the rerefresh
-		c.Assert(tasks[len(tasks)-1].Kind(), Equals, "check-rerefresh")
-		tasks = tasks[:len(tasks)-1] // and now forget about it
+		if scenario.update {
+			c.Check(tasks[len(tasks)-1].Kind(), Equals, "check-rerefresh")
+			tasks = tasks[:len(tasks)-1] // and now forget about it
+		}
```

Comment:
Same question about checking that "check-rerefresh" is not there.

Reply:
same because of counting